OctaneVSPlugin/View/DetailsToolWindowControl.xaml

```diff
@@ -650,83 +887,102 @@
+                                                            Value="Visible" />
                                                 </DataTrigger>
                                             </Style.Triggers>
                                         </Style>
                                     </DockPanel.Style>
 
                                     <Border BorderThickness="1,0,0,0"
-                                    BorderBrush="#FFC5C5C5"
-                                    Margin="3 15 3 0"
-                                    Width="1"
-                                    Background="White"/>
+                                            BorderBrush="#FFC5C5C5"
+                                            Margin="3 15 3 0"
+                                            Width="1"
+                                            Background="White" />
 
-                                    <Separator DockPanel.Dock="Top" Margin="0 15 0 15"/>
+                                    <Separator DockPanel.Dock="Top"
+                                               Margin="0 15 0 15" />
 
                                     <Grid>
                                         <Grid.RowDefinitions>
-                                            <RowDefinition Height="Auto"/>
-                                            <RowDefinition/>
+                                            <RowDefinition Height="Auto" />
+                                            <RowDefinition />
                                         </Grid.RowDefinitions>
-                                        <TextBlock Grid.Row="0" Margin="10 0 0 10" FontSize="16" FontWeight="Bold">Comments</TextBlock>
+                                        <TextBlock Grid.Row="0"
+                                                   Margin="10 0 0 10"
+                                                   FontSize="16"
+                                                   FontWeight="Bold">Comments</TextBlock>
 
                                         <TextBox  Margin="5 25 0 10"
-                                          Name="commentText"
-                                          HorizontalAlignment="Left"
-                                          FontSize="12"
-                                          MinWidth="250" MaxWidth="250"
-                                          MinHeight="19" MaxHeight="19"
-                                          AcceptsReturn="True"
-                                          Text="{Binding CommentText, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
-                                          Background="{DynamicResource VsBrush.Window}"
-                                          Foreground="{DynamicResource VsBrush.WindowText}"
-                                          BorderThickness="1">
+                                                  Name="commentText"
+                                                  HorizontalAlignment="Left"
```

Comment:
Here are so many hard coded values for the size of the comments - does this work correctly on multiple resolutions? e.g 1280x1024, 1650x1050

Reply:
Again this part of the code was not touched by me, hence I assume the one who wrote it knows how it works on multiple resolutions, nontheless I tested it out and it seems to keep proportions when on different resolutions